doc/examples/reconst_csd.py

```diff
@@ -98,18 +101,17 @@
 fvtk.rm(ren, response_actor)
 
 """
-However, using an FA threshold is not a very robust method. It is dependent on
-the dataset (non-informed used subjectivity), and still depends on the
-diffusion tensor (FA and first eigenvector), which has low accuracy at high
-b-value. Alternatively, one can calibrate the response function directly from
-the data according to [Tax2014]_. 
+Depending on the dataset, FA threshold may not be the best way to divine the
```

Comment:
From the Oxford English Dictionary:


"Divine, v. : 2. To make out by sagacity, intuition, or fortunate
conjecture (that is, in some other way than by actual information); to
conjecture, guess."

On Sun, Mar 15, 2015 at 4:49 PM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In doc/examples/reconst_csd.py
> <https://github.com/nipy/dipy/pull/594#discussion_r26454168>:
>
> > @@ -98,18 +101,17 @@
> >  fvtk.rm(ren, response_actor)
> >
> >  """
> > -However, using an FA threshold is not a very robust method. It is dependent on
> > -the dataset (non-informed used subjectivity), and still depends on the
> > -diffusion tensor (FA and first eigenvector), which has low accuracy at high
> > -b-value. Alternatively, one can calibrate the response function directly from
> > -the data according to [Tax2014]_.
> > +Depending on the dataset, FA threshold may not be the best way to divine the
>
> divine?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/594/files#r26454168>.
>